<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMultiVolumesUltrasoundModule</class>
 <widget class="qSlicerWidget" name="qSlicerMultiVolumesUltrasoundModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="qMRMLNodeComboBox" name="OIGTLConnectorNodeSelector">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLIGTLConnectorNode</string>
             </stringlist>
            </property>
            <property name="showHidden">
             <bool>false</bool>
            </property>
            <property name="showChildNodeTypes">
             <bool>false</bool>
            </property>
            <property name="addEnabled">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../qSlicerMultiVolumesUltrasoundModule.qrc">:/refund-arrows.black_full.jpeg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qMRMLNodeComboBox" name="MultiVolumeContainerNodeSelector">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLMultiVolumeNode</string>
             </stringlist>
            </property>
            <property name="showChildNodeTypes">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>true</bool>
            </property>
            <property name="renameEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="VolumeSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pauseVolumesButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding:0px;margin:0px;</string>
            </property>
            <property name="icon">
             <iconset resource="../qSlicerMultiVolumesUltrasoundModule.qrc">
              <normaloff>:/1335295004_media-pause.png</normaloff>:/1335295004_media-pause.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>15</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playVolumesButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding:0px;margin:0px;</string>
            </property>
            <property name="icon">
             <iconset resource="../qSlicerMultiVolumesUltrasoundModule.qrc">
              <normaloff>:/1335294988_media-play.png</normaloff>:/1335294988_media-play.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>15</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerMultiVolumesUltrasoundModule.qrc"/>
 </resources>
 <connections/>
</ui>
